### ISSUE TITLE
Better document fault/slab behavior near the poles in spherical coordinates

### ISSUE THREAD

**Issue by MFraters** [GeodynamicWorldBuilder/WorldBuilder#319]
Because the way the faults and slabs are defined and computed in spherical coordinates, the behavior of those features near the poles might not be what one would expect. The faults and slabs are defined to be perpendicular to the lines which defines them in the natural coordinates. This means that in spherical coordinates, a slab can not cross the pole, since all lines in spherical coordinates will cross the poles. 

In most cases this should not matter too much (there are no slabs on the poles on the earth), but there should be a warning and explanation in the manual. Providing the ability for slabs to cross the poles, would require switching to cartesian coordinates earlier in the process, but then you would lose some nice properties and introduce new issues (like not automatically interpolating between spherical coordinates and having the correct depth). 

So in summary, I think the current solution is the best one. There might be a better solution, but I would need to think about it and go through the relevant code. Since I don't think anyone is using this near the poles, improved documentation is for now the better solution I think.

![fault_spherical_coordinates](https://user-images.githubusercontent.com/7631629/125703810-fd619854-b361-422b-b1e1-452b053516f9.png)

This might be relevant for @alarshi and @bangerth.

**Comment by MFraters** [comment]
I think this is sovled, see #338
